.eslintcache

```diff
@@ -0,0 +1 @@
+[{"C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\App.js":"1","C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\components\\AddOrder.js":"2","C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\components\\EditOrder.js":"3","C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\components\\OrderForm.js":"4","C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\components\\Statuses.js":"5"},{"size":2920,"mtime":1634836767096,"results":"6","hashOfConfig":"7"},{"size":2576,"mtime":1634836767133,"results":"8","hashOfConfig":"7"},{"size":3362,"mtime":1634836767174,"results":"9","hashOfConfig":"7"},{"size":6114,"mtime":1634836767243,"results":"10","hashOfConfig":"7"},{"size":1580,"mtime":1634836948412,"results":"11","hashOfConfig":"7"},{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"gppgzb",{"filePath":"14","messages":"15","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"16","messages":"17","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"18","messages":"19","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"20","messages":"21","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\App.js",[],"C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\components\\AddOrder.js",[],"C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\components\\EditOrder.js",[],"C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\components\\OrderForm.js",[],"C:\\Projects\\us-frontend\\us-congress-pizza-flag-tracker-frontend\\src\\components\\Statuses.js",[]]
```

Comment:
Can you add this file to the .gitignore and then run git rm <file>

Reply:
done